cmd/plume/containerlinux.go

```diff
@@ -62,6 +62,7 @@ var (
 				"ap-southeast-2",
 				"ap-northeast-1",
 				"ap-northeast-2",
+				"af-south-1",
```

Comment:
Like in https://github.com/flatcar-linux/flatcar-build-scripts/pull/93, maybe add also "ap-southeast-3" and "eu-south-1"?

Reply:
Good catch, Done!